Essentials/src/main/java/com/earth2me/essentials/MobCompat.java

```diff
@@ -299,35 +292,4 @@ private Villager.Profession asEnum() {
             return RegistryUtil.valueOf(Villager.Profession.class, newProfession, oldProfession);
         }
     }
-
-    public enum BoatVariant {
```

Comment:
Does this drop support for non-oak boats below 1.21.3?

Reply:
did not test